README.md

```diff
@@ -204,6 +204,61 @@ Logbook comes with some built-in strategies:
 - [`StatusAtLeastStrategy`](logbook-core/src/main/java/org/zalando/logbook/core/StatusAtLeastStrategy.java)
 - [`WithoutBodyStrategy`](logbook-core/src/main/java/org/zalando/logbook/core/WithoutBodyStrategy.java)
 
+### Attribute Extractor
+Starting with version 3.4.0, Logbook is equipped with a feature called *Attribute Extractor*. Attributes are basically a
+list of key/value pairs that can be extracted from request and/or response, and logged with them. The idea was sprouted
+from [issue 381](https://github.com/zalando/logbook/issues/381), where a feature was requested to extract the subject
+claim from JWT tokens in the authorization header.
+
+The `AttributeExtractor` interface has two `extract` methods: One that can extract attributes from the request only, and
+one that has both request and response at its avail. The both return an instance of the `HttpAttributes` class, which is
+basically a fancy `Map<String, Object>`. Notice that since the map values are of type `Object`, they should have a 
+proper `toString()` method in order for them to appear in the logs in a meaningful way.

```

Comment:
If we rely on `toString()` anyway, why don't we change the signature to `Map<String, String>` then to be very precise in what we support?

Reply:
Good point.
Would it make sense to rely on `toString()` for some log formatters (e.g., `DefaultHttpLogFormatter`), but some others like `JsonHttpLogFormatter` can another mechanism (say ObjectMapper) to convert objects to string representation?

This gives the implementers of AttributeExtractor the flexibility of not having to deal with Strings if they know that the log formatter they are going to use takes care of it.

Reply:
Please also see 0725b94.